app/models/concerns/statusable.rb

```diff
@@ -20,10 +20,13 @@ module Statusable
     dropoff: { key: I18n.t('patient.status.key.dropoff'),
                help_text: I18n.t('patient.status.help.dropoff')},
     resolved: { key: I18n.t('patient.status.key.resolved', fund: ActsAsTenant.current_tenant&.name),
-                help_text: I18n.t('patient.status.help.resolved')}
+                help_text: I18n.t('patient.status.help.resolved')},
+    completed: { key: I18n.t('patient.status.key.completed'),
+                 help_text: I18n.t('patient.status.help.completed') },
```

Comment:
What do we use `help_text` for? I wasn't able to find a reference

Reply:
These show up in the Status tooltip on patient records!

<img width="334" alt="image" src="https://github.com/DARIAEngineering/dcaf_case_management/assets/3866868/0fe6e575-6201-4eb4-a13d-1f04ade4d77f">